Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
 gem "jekyll", "4.2.0"
+
+gem "webrick", "~> 1.7"
```

Comment:
is webrick needed for production ?

Reply:
My local installation wasn't working. It worked in Docker, but I had to install webrick to make it work in local. As the error said, it was missing webrick module. 

Here's the error I had got. 

```

Configuration file: /Users/aswinbehera/Experiences/SSR/envoyproxy.github.io/_config.yml
            Source: /Users/aswinbehera/Experiences/SSR/envoyproxy.github.io
       Destination: /Users/aswinbehera/Experiences/SSR/envoyproxy.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
                    done in 0.165 seconds.
 Auto-regeneration: enabled for '/Users/aswinbehera/Experiences/SSR/envoyproxy.github.io'
                    ------------------------------------------------
      Jekyll 4.2.0   Please append `--trace` to the `serve` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/Users/aswinbehera/Experiences/SSR/envoyproxy.github.io/vendor/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/commands/serve/servlet.rb:3:in `require': cannot load such file -- webrick (LoadError)
        from /Users/aswinb
```
and more cascading errors below, which I haven't added.

Reply:
my feeling is that the changes to the web should be doable separate to any version/dependency changes

im not against changing the versions, just would rather that be done separately

Reply:
So should I remove the webrick entry from the gemfile. What should be my next change?

Reply:
i would remove all version changes/additions and see how it builds in CI

Reply:
Cool. let me try it and push the changes.